online-rest-api/src/main/java/eu/itesla_project/online/rest/api/ProcessApiService.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.online.rest.api;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+/**
+ *
+ * @author Giovanni Ferrari <giovanni.ferrari@techrain.it>
+ */
+public abstract class ProcessApiService {
```

Comment:
It was automatically generated starting from a "swagger" definition of the REST API .